backend/templates/index.html

```diff
@@ -151,85 +151,94 @@ <h4 style="margin-top: 0px;">QLever UI Shortcuts:</h4>
+                      <button class="btn btn-default" onclick="window.location=location.protocol + '//' + location.host">
+                        <i class="glyphicon glyphicon-trash"></i> Reset
+                      </button>
+                      <button class="btn btn-default" onclick="processQuery(-1, this).catch(error => log(error.message, 'requests'));">
+                        <i class="glyphicon glyphicon-refresh"></i> Clear cache
+                      </button>
+                      <button class="btn btn-default" onclick="showQueryPlanningTree();">
+                        <i class="glyphicon glyphicon-object-align-vertical"></i> Analysis
+                      </button>
+                      <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown">
+                        <i class="glyphicon glyphicon-align-left"></i> Examples
+                        <span class="caret"></span>
+                      </button>
+                      <ul class="dropdown-menu">
+                        {% for example in examples %}
+                          <li>
+                            <a onclick="example=1;editor.setValue(examples[{{ forloop.counter0 }}]);">
+                              {% if request.user.is_authenticated %}
+                              <span onclick="window.open('/admin/backend/example/{{ example.pk }}/change/','_blank');" style="float: right;">
+                                <i class="glyphicon glyphicon-pencil"></i>
+                              </span>
+                              {% endif %}
+                              <span style="margin-right: 30px">{{ example.name }}</span>
+                            </a>
+                          </li>
+                        {% endfor %}
+                        {% if request.user.is_authenticated %}
+                          <li class="divider"></li>
+                          <li>
+                            <a onclick="window.open('/admin/backend/example/add/?backend={{ backend.pk }}&query='+encodeURIComponent(editor.getValue()),'_blank');">
+                              <i class="glyphicon glyphicon-check"></i> Add current query as example
+                            </a>
+                          </li>
+                          <li>
+                            <a href="/admin/backend/example/add/?backend={{ backend.pk }}" target="_blank">
+                              <i class="glyphicon glyphicon-edit"></i> Add new example
+                            </a>
+                          </li>
+                        {% endif %}
+                      </ul>
+                    </div>
+                  </div>
+                  <div class="col-md-3" style="margin-top:5px">
+                    <button id="formatButton" class="btn btn-success" style="min-width: 7.857em;">
+                      <i class="glyphicon glyphicon-align-left"></i> <span>Format</span>
+                    </button>
+                  </div>
```

Comment:
The `min-width` stems from the execute button and the rest of the changes here are indentation, right?

Reply:
Yes, exactly.
I cleaned up the whole grid situation in this section a bit.